File: source/Web/app/Models/Order.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsToMany;

class Order extends Model
{

    protected $table = 'Order';
    public $timestamps = false;

    protected $fillable = [
        'name', 'address', 'email', 'phone', 'status', 'is_paid', 'product_price', 'ship_price', 'user_id', 'admin_id', 'bank_code', 'transaction_code', 'created_at',
    ];

    public function orderProducts(): BelongsToMany
    {
        return $this->belongsToMany(Product::class, 'OrderProduct')->withPivot([
            'name',
            'amount',
            'price',
            'discount_price',
            'original_price',
        ]);
    }
}

File: source/Webform/Models/Order.cs
namespace Webform.Models
{
    using System;
    using System.Collections.Generic;
    
    public partial class Order
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Order()
        {
            this.OrderProducts = new HashSet<OrderProduct>();
        }
    
        public long id { get; set; }
        public string name { get; set; }
        public string address { get; set; }
        public string email { get; set; }
        public string phone { get; set; }
        public int status { get; set; }
        public bool is_paid { get; set; }
        public double ship_fee { get; set; }
        public double product_price { get; set; }
        public double ship_price { get; set; }
        public Nullable<long> user_id { get; set; }
        public Nullable<long> admin_id { get; set; }
        public string bank_code { get; set; }
        public string transaction_code { get; set; }
        public System.DateTime created_at { get; set; }
    
        public virtual Admin Admin { get; set; }
        public virtual User User { get; set; }
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<OrderProduct> OrderProducts { get; set; }
    }
}
